llvm/test/CodeGen/SPIRV/extensions/SPV_INTEL_subgroups/cl_intel_subgroup_local_block_io.ll

```diff
@@ -0,0 +1,156 @@
+; Modified from: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/test/extensions/INTEL/SPV_INTEL_subgroups/cl_intel_sub_groups.ll
```

Comment:
Apologies, but what is a principal difference between this test and lets say https://github.com/llvm/llvm-project/blob/main/llvm/test/CodeGen/SPIRV/extensions/SPV_INTEL_subgroups/cl_intel_sub_groups.ll ? From the first glance they seem to test the same functionality, though the naming of cl_intel_sub_groups.ll is missleading.

Reply:
This test does not have shuffle instructions. 
But the actual main difference is that input pointer to the builtins is in Local (`addrspace (3)`) instead of Global (`addrspace (1)`). `cl_intel_subgroup_local_block_io` just allows to use local ptr instread of global ptr, so no different functionality is expected to be tested except address space